compiler/rustc_lint/messages.ftl

```diff
@@ -50,6 +50,9 @@ lint_deprecated_lint_name =
 lint_renamed_or_removed_lint = {$msg}
     .suggestion = use the new name
 
+lint_suspicious_double_ref_op =
+    using `.{$call}()` on a double reference, which returns `{$ty}` instead of {$op} the inner type
```

Comment:
```suggestion
    using `.{$call}()` on a double reference, which returns `{$ty}` instead of {$op}, the inner type
```

Correct me (a non-native speaker of english) if I'm wrong, but I think there needs to either be a comma, or it needs to be reodered `instead of the inner type {$op}`. I think it's better to have the two types close to each other, so I prefer the variant with the comma.

Reply:
`$op` is not a type. It is either "dereferencing", "borrowing", or "cloning" depending on the trait.

Reply:
I see. Those words should also be in the ftl file then however.

Reply:
oh right.. hmmmm. How should I do that

Reply:
I suppose they don't have to? Since https://projectfluent.org/ shows that you can match on variables.

Reply:
I think at the very least you should list it explicitly, so that translators know that there is only these options. From what I understand, the translation system was designed so that translators do not have to look at the source code at all.

```
{$op ->
    *[should_not_happen] [{$op}]
    [deref] dereferencing
    [borrow] borrowing
    [clone] cloning
}
```

How to pair that up with enums, I don't really know.